wg-advisors/why/why_branding_licencing.md

```diff
@@ -1,13 +1,68 @@
+used by all Apache projects. The license is the Apache License, Version 2.0, and the branding is based
+on the project name. The licence is a permissive open source license that allows the software to be used
+by anyone for any purpose, and the branding is a set of rules that ensure that the software is recognized
+as an Apache project and that the project is recognized as part of the Apache Software Foundation.
+
+The software that ASF projects produce is released under the Apache License, Version 2.0, which is a permissive
+open source license that allows the software to be used by anyone for any purpose and it is important that
+we do not limit anyone from using the software in any way. And while we allow anyone to use the software,
+and redistribute it, we do not allow anyone to claim that they are the "owners" of the software, or that
+they are controlling the project.
+
+This is why it is important to ensure that the project's brand is not used in a way that could cause confusion
+or mislead users on who is owning or controlling the project. This includes ensuring that the project's name
+is not used in a way that could imply that another party is controlling the project - assuring
+vendor neutrality.
+
+Some, popular, Apache projects have a strong brand recognition, and it's important to ensure that the
+brand is used consistently across all project's assets, including the website, documentation, and
+marketing materials and the ASF registers the project's name as a trademark to ensure that the project's
+brand is protected.
+
+The ASF has a set of policies that define how the Apache brand should be used, and it's important that
+all projects follow these guidelines to ensure that the brand is used consistently and correctly.
+
+It's a PMC responsibility to ensure that the project's brand is used correctly and to take action if the
+brand is used in a way that could cause confusion or mislead users.
+
+The foundation has legal ways to enforce the brand and license, but it's always better to resolve any
+issues informally, by contacting the party that is using the brand incorrectly and asking them to correct
+the issue.
+
+Events are a special case, as they are often organized by third parties, and it's important to ensure
+that the event is recognized as an Apache event and that the event is organized in a way that is consistent
+with the Apache Software Foundation's values and principles.
+
 ## Links to relevant documents
 
+* [The Apache Software Foundation Trademark Policy](https://www.apache.org/foundation/marks/)
+* [The Apache Licences](https://apache.org/licenses/)
+* [Third party event branding policy](https://www.apache.org/foundation/marks/events.html)
+
 ## Why are we doing it?
 
+The software that ASF projects produce is released under the Apache License, Version 2.0, which is a permissive
+open source license that allows the software to be used by anyone for any purpose and it is important that
+we do not limit anyone 
```

Comment:
This, again, repeats the first paragraph.

Reply:
Right. Removed it.